src/modules/index.js

```diff
@@ -0,0 +1,132 @@
+const containerElement = document.querySelector('.container-todo');
+const ulElement = document.querySelector('.ul-element');
+
+const Tasks = class {
+  constructor(description, completed = false, index = 2) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  populateFields = () => {
+    let i = -1;
+    this.tasks.forEach((task) => {
+      i += 1;
+      task.index = i;
+    });
+    localStorage.setItem('savedTasks', JSON.stringify(this.tasks));
+  };
+
+  removeTask(task) {
+    const result = this.tasks.filter((b) => b !== task);
+    this.tasks = result;
+    this.populateFields();
+  }
```

Comment:
- [ ] I am not able to remove a task because I am not able to add please fix that
- [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

Reply:
- [ ] Please implement function for editing task descriptions